pyanaconda/ui/gui/helpers.py

```diff
@@ -124,6 +126,68 @@ def input_confirmation(self):
         """
         pass
 
+    def input_screen_about_to_be_displayed(self):
+        """The screen is about to be entered."""
+        # The screen containing then checked input fields is about to be entered,
+        # so remember the initial content of the fields so that we can detect if
+        # the content has changed.
+        self.initial_input = self.input
+        self.initial_input_confirmation = self.input_confirmation
+
+    @property
+    def initial_input(self):
+        """Content of the input field when the screen was entered.
+
+        And before user could influence it's contents.
```

Comment:
This should be standalone not something what is connecting to the name of the docstring. In other words, please remove the `And` part or remove this completely.